logging/log_collection_forwarding/logging-output-types.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-output-types"]
+= Log output types
+:context: logging-output-types
+
+toc::[]
+
+Log outputs specified in the `ClusterLogForwarder` CR can be any of the following types:
```

Comment:
This information is also in [About forwarding logs to third-party systems](https://64935--docspreview.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/log-forwarding#cluster-logging-collector-log-forwarding-about_log-forwarding). Dies it need to be in both places? Use a snippet to make sure the info is the same in both places?

Reply:
I'd say it's out of scope for this PR. I didn't add new content really around outputs, just moved it / cleaned it up, and we're in the middle of a big content re-org now, so this will be fixed under another PR. We also have separate Jiras related to output docs to follow up on.